dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/metadata/WebModuleMetaData.java

```diff
@@ -83,6 +83,24 @@ public interface WebModuleMetaData extends ModuleMetaData
      
      void setAnnotatedSecurityMetaData(Object metaData);
      
+     /**
```

Comment:
This interface is an SPI.  So the user / stack product can use it as well in user features.